sstables/sstable_set.cc

```diff
@@ -175,6 +175,11 @@ sstable_set::erase(shared_sstable sst) {
     _impl->erase(sst);
 }
 
+size_t
+sstable_set::size() const {
```

Comment:
please define as noexcept so it can be safely used in exception sensitive paths.